packages/app/src/cli/services/context.ts

```diff
@@ -584,16 +570,32 @@ async function getAppDevCachedContext({
   directory,
   token,
   configName,
+  commandConfig,
 }: {
   reset: boolean
   directory: string
   token: string
   configName?: string
+  commandConfig: Config
 }): Promise<AppDevCachedContext> {
+  const previousCachedInfo = getAppInfo(directory)
+
   if (reset) clearAppInfo(directory)
 
   let cachedInfo = getAppInfo(directory)
 
+  if (previousCachedInfo === undefined) {
+    const explanation =
+      `\nLooks like this is the first time you're running dev for this project.\n` +
+      'Configure your preferences by answering a few questions.\n'
+    outputInfo(explanation)
+  }
```

Comment:
I think this message can go, we can also remove the similar one that says "Looks like this is the first time you're running generate for this project."

Reply:
I'd rather do it here, because the message that we print when running `generate` for the first time will be incorrect, no?